### PR TITLE
scripts: fix shellcheck errors

### PR DESCRIPTION
**Pull request by mythi**
Signed-off-by: Mikko Ylinen <mikko.ylinen@intel.com>

### CODE REVIEW

**Review by rojkov — DISMISSED** [review]

File: scripts/set-image-pull-policy.sh [hunk 1]
```diff
@@ -18,8 +18,8 @@ fi
 
 IMAGE_PULL_POLICY=$1
 
-echo IMAGE_PULL_POLICY=$IMAGE_PULL_POLICY
+echo IMAGE_PULL_POLICY="$IMAGE_PULL_POLICY"
 
-sed -i -e "s;\(imagePullPolicy\:\ \).*;\1$IMAGE_PULL_POLICY;" $(git grep -l 'imagePullPolicy' deployments/*.yaml demo/*.yaml)
-
-sed -i -e "s;\(ImagePullPolicy\:\ \).*;\1\"$IMAGE_PULL_POLICY\",;" $(git grep -l 'ImagePullPolicy' pkg/controllers/*/*.go)
```
Comment (bart0sh) [inline scripts/set-image-pull-policy.sh:25]: pkg/controllers/*/*.go is not processed in the new version. It doesn't look like a fix fot shellcheck errors to me.

Comment (bart0sh) [reply scripts/set-image-pull-policy.sh:25]: it's ok to have it here if it's done intentionally.

**Review by bart0sh — APPROVED** [review]
/lgtm

File: scripts/set-image-pull-policy.sh (hunk 1 above)
Comment (mythi) [reply scripts/set-image-pull-policy.sh:25]: yes it's intentional

**Review by rojkov — APPROVED** [review]

**Pull request merged**